docs/template-context.md

```diff
@@ -0,0 +1,41 @@
+---
```

Comment:
A few notes, @Florence-Njeri 😄✌🏽:
- Remember to be **consistent**. Since you have not capitalized the `generator` tool, let's stick to writing it in lowercase.
- I know sometimes it can feel confusing, wondering whether or not to capitalize words. (Ex: object, parser, etc.) But you should **only capitalize the first word of a sentence and for all proper nouns** _(words that name a specific person, place, organization, or thing)_

Reply:
Alright. Thank you